docs/reference/indices/shrink-index.asciidoc

```diff
@@ -28,28 +28,36 @@ POST /twitter/_shrink/shrunk-twitter-index
 Before you can shrink an index:
 
 * The index must be read-only.
-* A copy of every shard in the index must reside on the same node.
-* The <<cluster-health, cluster health>> status must be green.
+* All primary and replica shards for the index must reside on the same node.
```

Comment:
I believe that only the primary shards must reside on the same node. ES won't allocate a replica shard to a node containing its primary since that would serve no purpose.